### ISSUE TITLE
Is it possible to do hyperparameter search when finetuning a pre-existing model?

### ISSUE THREAD

**Issue by geniki** [flairNLP/flair#2077]
I'd like to use the hyperparameter search functionality (` SequenceTaggerParamSelector`) with the initial model being one of the models you already provide. I don't want to change the architecture, just search for things such as optimal learning rate.

Is the above possible? Or does hyperparameter search always start from a random model?

**Comment by afi1289** [comment]
Hi,
Did you find a solution?
